.github/wiki/Bitrise.md

```diff
@@ -0,0 +1,64 @@
+# Bitrise
+Use the Bitrise template to start a new project with Bitrise as the CI/CD tool.
+
+## Workflows and Steps
+
+Out of the box, the Bitrise Template has the following workflows and steps:
+
+| test                      | deploy_app_store                                        | deploy_staging                          | deploy_release_firebase                   |
+|---------------------------|---------------------------------------------------------|-----------------------------------------|-------------------------------------------|
+| Git Clone Repository      | Git Clone Repository                                    | Git Clone Repository                    | Git Clone Repository                      |
+| Bitrise.io Cache:Pull     | Bitrise.io Cache:Pull                                   | Bitrise.io Cache:Pull                   | Bitrise.io Cache:Pull                     |
+| Run CocoaPods install     | Run CocoaPods install                                   | Run CocoaPods install                   | Run CocoaPods install                     |
+| Fastlane - Build and Test | Xcode Test for iOS                                      | Xcode Test for iOS                      | Xcode Test for iOS                        |
+| Fastlane - Run XCov       | Fastlane Match                                          | Fastlane Match                          | Fastlane Match                            |
+| Danger                    | Fastlane - Build and Upload Production App to App Store | Fastlane - Build and Upload Staging App | Fastlane: Build and Upload Production App |
+
+## Trigger Map
+
+| Workflow                | Trigger                 |
+|-------------------------|-------------------------|
+| test                    | Create or Update a PR   |
+| deploy_staging          | Push branch `develop`   |
+| deploy_release_firebase | Push branch `release/*` |
+| deploy_app_store        | Push branch `master`    |
```

Comment:
I think we can add the `main` branch also'
```suggestion
| deploy_app_store        | Push branch `master`/`main`  |
```

Reply:
Currently, in `bitrise.yml`, we don't have the trigger for the `main` branch. I think it is better to create a new issue to add a trigger for `main` as well as clean up unused env variables then I will update the document again.